### PR TITLE
Completed html-relative-links exercise - Francis Manalang

### PR DESCRIPTION
**Pull request by francismanalang**
https://user-images.githubusercontent.com/101234537/176250340-c0936d17-99f3-4c5d-b7ab-65a6b6ea216e.mov

### CODE REVIEW

File: html-relative-links/projects/projects.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    img {
+      width: 300px;
+    }
+  </style>
+</head>
+<body>
+  <ul>
+    <li><a href="../index.html">Home Page</a></li>
+    <li><a href="../about-me/about-me.html">About Me</a></li>
+    <li><a href="../projects/projects.html">Projects</a></li>
```
Comment (codingcodymiller) [inline html-relative-links/projects/projects.html]: In this path, you are exiting the current directory just to go right back into it, which is unnecessary. You can just state the name of the file you want to navigate to.

File: html-relative-links/contact-us/contact-us.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+      width: 300px;
+    }
+  </style>
+</head>
+<body>
+  <ul>
+    <li><a href="../index.html">Home Page</a></li>
+    <li><a href="../about-me/about-me.html">About Me</a></li>
+    <li><a href="../projects/projects.html">Projects</a></li>
+    <li><a href="../contact-us/contact-us.html">Contact Us</a></li>
```
Comment (codingcodymiller) [inline html-relative-links/contact-us/contact-us.html]: In this path, you are exiting the current directory just to go right back into it, which is unnecessary. You can just state the name of the file you want to navigate to.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: html-relative-links/contact-us/contact-us.html (hunk 2 above)
Comment (codingcodymiller) [reply html-relative-links/contact-us/contact-us.html]: MIssed a spot.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: html-relative-links/contact-us/contact-us.html (hunk 2 above)
Comment (francismanalang) [reply html-relative-links/contact-us/contact-us.html]: got it. I have made the changes and committed. do I need to make another pul request?

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**